be/tool/beck.c

```diff
@@ -1584,13 +1799,29 @@ static void builder_done(struct m0_be_tx_bulk   *tb,
 			 uint64_t                worker_index,
 			 uint64_t                partition)
 {
+	struct action          *act;
+	struct worker_off_info *winfo;
 
+	act = user;
+	if (act != NULL) {
+		winfo = &off_info.oi_offset[worker_index];
+		winfo->woi_offset[partition] = act->a_node_offset;
+		winfo->woi_act_done[partition]++;
+		m0_mutex_lock(&off_info.oi_part_lock[partition]);
+		off_info.oi_pinfo.pi_act_done[partition]++;
+		m0_mutex_unlock(&off_info.oi_part_lock[partition]);
+
+		if (!(winfo->woi_act_done[partition] % NV_OFFSET_SAVE_ACT_DELTA))
```

Comment:
According to coding-style, checking int should like this:
```
if (winfo->woi_act_done[partition] % NV_OFFSET_SAVE_ACT_DELTA == 0)
```

Reply:
updated in commit #c88f74